src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,55 @@
+package lotto.view;
+
+import java.util.List;
+import lotto.domain.Lotto;
+import lotto.domain.LottoNumber;
+import lotto.domain.LottoResult;
+import lotto.domain.Rank;
+
+public class OutputView {
+
+    public void printLottos(List<Lotto> lottos) {
+        System.out.println();
+
+        String description = String.format("%d개를 구매했습니다.", lottos.size());
+        System.out.println(description);
+
+        lottos.forEach(this::printLotto);
+    }
+
+    private void printLotto(Lotto lotto) {
+        List<LottoNumber> numbers = lotto.getNumbers();
+        List<Integer> values = numbers.stream()
+                .map(LottoNumber::getNumber)
+                .toList();
+        System.out.println(values);
+    }
```

Comment:
```suggestion
     private void printLotto(Lotto lotto) {
         List<Integer> numbers = lotto.getNumbers().stream()
                 .map(LottoNumber::getNumber)
                 .toList();
         System.out.println(numbers);
  }
```
values가 없어도 될 것 같아요!

Reply:
오, 일부로 values를 변수로 선언하신 걸까요?
이유가 궁금해요!